gateway/1.4/modules/ROOT/pages/flex-local-reg-run-up.adoc

```diff
@@ -36,7 +36,14 @@ include::partial$task-reg-run-flex-gateway.adoc[tags=sub-coll-info;user-replace-
 //reg command
 include::partial$task-reg-run-flex-gateway.adoc[tags=reg-command-heading;reg-command-1;docker-reg-command;user-reg-command;environment-reg-command;organization-reg-command;output-reg-command-docker;reg-command-2;after-reg;k8s-after-reg]
 //install helm chart
-include::partial$task-reg-run-flex-gateway.adoc[tags=k8s-install-helm-chart]
```

Comment:
needed to break out tag `k8s-install-helm-chart` here and elsewhere to include the Helm chart info with the version include so that version number is updated in only one place: include::partial$prerequisites.adoc[tag=helm] 

we keep all our version numbers in partial$prerequisites.adoc